### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by JimAppreciator512** [theabbie/ifunny#8]
Hello nice repo, I run an ifunny discord bot that could use this actually. But I have some questions about the API and it's functionality:

- do you require an account to get an oauth token, or can you just do stuff without
- how did you figure what the URL was?
- is there functionality for user search
- is there functionality to look through a user's subscriptions and such, like profile info
- is there a website that displays all the functionality

If you ever see this or ever read this, thanks for your time and nice work.

**Comment by theabbie** [comment]
@JimAppreciator512 
1. Yes, account is needed to post, getting memes may not require account but this library needs it for all functions.
2. It was extract from their Android app through network interception.
3. Not in this library, if it's in the mobile app it should be possible.
4. Same as 3rd.
5. No, only the README of this repo, and all functionalities in Mobile app can be made possible but may not be present in this library.

Thanks, I am not sure if this library still works but the code may be useful.

**Comment by JimAppreciator512** [comment]
Yeah that's what I hate about iFunny. I tried recently to intercept the traffic from my phone because for some reason I couldn't emulate iFunny.

Do you mind telling me how you intercepted the traffic from your phone? Wireshark?

**Comment by theabbie** [comment]
@JimAppreciator512 there is an app HTTPCanary for it, it works only if device has root access.
